src/Marten.Testing/Linq/Compatibility/Support/TargetSchemaFixture.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Marten.Testing.Linq.Compatibility.Support
+{
+
+    public abstract class TargetSchemaFixture : IDisposable
+    {
+        public readonly Target[] Documents = Target.GenerateRandomData(1000).ToArray();
```

Comment:
Sidenote: wouldn't 100 be enough and make tests run faster?

Reply:
Dunno, but it's been pretty fast with 1K